code/game/objects/items/devices/chameleonproj.dm

```diff
@@ -36,7 +36,7 @@
 	if(!check_sprite(target))
 		return
 	if(!active_dummy)
-		if(isitem(target) && !istype(target, /obj/item/disk/nuclear))
+		if(!isturf(target) && !istype(target, /obj/structure/falsewall))
```

Comment:
Why not isobj, or do you want them to be able to copy mobs as well?

Reply:
yes but I wanted it to be megafauna so let me remove carbons.